quickstarts/post-configure-example/README.md

```diff
@@ -0,0 +1,200 @@
+# JBoss CLI PostConfigure example
+
+This example will explain how to create additional JBoss EAP users using the Post Configure feature available on the 
+Container images based on JBoss EAP.
+
+The steps described on this quickstart will work on the following container images:
+
+- RHPAM KIE Server
+- RHDM KIE Server
+- Business Central
+- Business Central Monitoring
+- Decision Central
+- Dashbuilder
+- RHPAM/DM Controller
+
+
+Note that, this procedure can be applied to any kid of post configure actions.
+
+## Preparing the needed files
+
+Before moving forward with the deployments, let's prepare the needed files.
+You need to have the following files:
+
+- [add-users.cli](add-users.cli): JBoss CLI Batch script, your script must be added between the commands below:
+  - ```bash
+    embed-server --std-out=echo  --server-config=standalone-openshift.xml
+    batch
+
+    <your jboss-cli commands>
+
+    run-batch
+    quit
+    ```
+    
+- [delayedpostconfigure.sh](delayedpostconfigure.sh): This empty file is needed until the following jira is not fixed:
+  https://issues.redhat.com/browse/RHPAM-3665
+
+- [postconfigure.sh](postconfigure.sh): The script that will actually call the JBoss-CLI script.
+
+
+With these 3 files in place, the next step will be creating a config-map and mount it under `/opt/eap/extensions` directory
+on the target container image(s).
+
+```bash
+$ oc create configmap postconfigure \
+    --from-file=add-users.cli=add-users.cli \
+    --from-file=delayedpostconfigure.sh=delayedpostconfigure.sh \
+    --from-file=postconfigure.sh=postconfigure.sh
+configmap/postconfigure created
```

Comment:
Is created message needed to be mentioned here? It may be accidentally copied and used as a command.

Reply:
It can be good to have, users can compare their results if something goes wrong. 
wdyt?

Reply:
it can stay, also if users copied it and put it to cmd line. The first command will pass and second one will throw some exception like file or directory does not exists.